### PR TITLE
Add LayoutWrapper Component

### PR DESCRIPTION
**Pull request by ronanmoris**
Created a simple wrapper with the AppBar on top.

### CODE REVIEW

**Comment by giubueno** [comment]
Hi @ronanmoris, please mark your pull request with the label "ready for review", please.

File: app/containers/EventsPage/Loadable.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  *
- * Asynchronously loads the component for TobBar
+ * Asynchronously loads the component for EventsPage
```
Comment (giubueno) [inline app/containers/EventsPage/Loadable.js]: I think this comment was right.

File: app/components/LayoutWrapper/tests/index.test.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**
  *
- * Tests for TobBar
+ * Tests for LayoutWrapper
```
Comment (giubueno) [inline app/components/LayoutWrapper/tests/index.test.js]: I think you changed the wrong file.

File: app/containers/EventsPage/Loadable.js (hunk 1 above)
Comment (ronanmoris) [reply app/containers/EventsPage/Loadable.js]: TopBar was the component we had created together and deleted. The component doesn`t exist anymore.

File: app/components/LayoutWrapper/tests/index.test.js (hunk 2 above)
Comment (ronanmoris) [reply app/components/LayoutWrapper/tests/index.test.js]: We deleted TobBar together.

**Pull request merged**